pandas/_libs/tslibs/offsets.pyx

```diff
@@ -300,7 +300,7 @@ cdef _validate_business_time(t_input):
 _relativedelta_kwds = {"years", "months", "weeks", "days", "year", "month",
                        "day", "weekday", "hour", "minute", "second",
                        "microsecond", "nanosecond", "nanoseconds", "hours",
-                       "minutes", "seconds", "microseconds"}
+                       "minutes", "seconds", "milliseconds", "microseconds"}
```

Comment:
i realize for consistency we also need 'millisecond' as well to directly set the value.

Reply:
To support the replacement usage, we would depend on `dateutil` to support it as well. I have reported the issue here: https://github.com/dateutil/dateutil/issues/1175. Should we block this until that's supported?

Reply:
why is this dependent on dateutil?

Reply:
> Should we block this until that's supported?

no

Reply:
if its not easy to do then i would suggest accepting but then raising NotImplementedError as its weird that its inconsistent.

Reply:
> why is this dependent on dateutil?

It eventually gets to this line: https://github.com/pandas-dev/pandas/blob/72327f32e2328d3e13b6c55617d71036fccdd0e1/pandas/_libs/tslibs/offsets.pyx#L323, which constructs a `dateutils.relativedelta`.

Reply:
I have added the line to check and raise `NotImplementedError` here.